ports/zephyr-cp/cptools/build_circuitpython.py

```diff
@@ -219,13 +219,13 @@ async def build_circuitpython():
     circuitpython_flags.append(f"-DCIRCUITPY_ENABLE_MPY_NATIVE={1 if enable_mpy_native else 0}")
     circuitpython_flags.append(f"-DCIRCUITPY_FULL_BUILD={1 if full_build else 0}")
     circuitpython_flags.append(f"-DCIRCUITPY_USB_HOST={1 if usb_host else 0}")
-    circuitpython_flags.append(f'-DCIRCUITPY_BOARD_ID=\\"{board}\\"')
+    circuitpython_flags.append(f"-DCIRCUITPY_BOARD_ID='\"{board}\"'")
```

Comment:
This looks good to me. I am just curious. Why doesn't
```py
    circuitpython_flags.append(f"-DCIRCUITPY_BOARD_ID={board}")
```
work here, since we know the `board` has no spaces? Does the context in which this is used require a double-quoted argument?

Similarly elsewhere below, where there are filenames in quotes, etc.